src/Controllers/Scheduler/allShows.php

```diff
@@ -11,7 +11,7 @@
 use \MyRadio\ServiceAPI\ServiceAPI;
 use \MyRadio\ServiceAPI\MyRadio_Show;
 
-$shows = MyRadio_Show::getAllShows();
+$shows = MyRadio_Show::getAllShows(1, true); # show_type_id=1, current_term_only=true
```

Comment:
use // and put it on the line above for clarity. Also it is clearer to say something in english rather than computer eg:
Get public shows (type 1) from this term only (true)